docs/language/functions.rst

```diff
@@ -353,6 +353,12 @@ values. Here is an example of an overloaded function:
 In the function foo, abs() is called with an ``int64`` so the second implementation
 of the function abs() is called.
 
+.. note::
+    Some target runtimes require unique function names (e.g. Substrate). In that case,
```

Comment:
I would simplify this to "On Substrate, public functions must be unique, so they cannot be overloaded. "..

Reply:
Sure. IMO `To fulfill this requirement,` is superfluous though.